modules-detection.mjs

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env zx
+
+const { ENV0_PR_SOURCE_BRANCH, ENV0_PR_TARGET_BRANCH, ENV0_TEMPLATE_PATH } = process.env;
+
+const sourceRef = ENV0_PR_SOURCE_BRANCH;
+const targetRef = ENV0_PR_TARGET_BRANCH;
+const workingDir = ENV0_TEMPLATE_PATH || ''
+
+await detect();
+
+async function detect() {
+    if (!sourceRef || !targetRef) {
+        console.log(`Skipping - can't detect source or target refs. source [${sourceRef}] target [${targetRef}]`);
+        return;
+    }
+
+    const { stdout: fileChangesRaw } = await $`git diff --name-only  ${sourceRef}..${targetRef}`;
```

Comment:
you installed that lib in our image? ( the one enabling you to $`command` ?

Reply:
Nope. I am installeing it part of the plugin https://github.com/env0/env0-terragrunt-modules-detection-plugin/blob/8366ea8c05362b615fa8500162858f1abeb6fdd7/env0.plugin.yml#L5